.eslintrc.json

```diff
@@ -37,6 +37,7 @@
     "import/no-unresolved": 0,
     "no-await-in-loop": 0,
     "arrow-body-style": 0,
-    "no-loop-func": 0
+    "no-loop-func": 0,
+    "arrow-parens": 0
```

Comment:
Had to disable rule, as almost all usages of functions with one argument are without parens

Reply:
I agree, that looks nicer